_drafts/embedded-world-recap.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Exciting things we saw at Embedded World 2024
```

Comment:
I'm pondering other options for this title 🤔 I like how low-key it is honestly but maybe we can hype it up more